hazelcast/src/hazelcast/client/compact.cpp

```diff
@@ -1495,6 +1497,7 @@ init_fp_table()
     }
     return FP_TABLE;
 }
+
```

Comment:
I think that this is also incorrect `return static_cast<int64_t>(fingerPrint);` isn't it? see [this](https://stackoverflow.com/questions/7601731/how-does-one-safely-static-cast-between-unsigned-int-and-int)

Reply:
Fixed.